scripts/install

```diff
@@ -93,199 +73,164 @@ ynh_script_progression --message="Storing installation settings..." --weight=7
 # STORE THE CONFIG FILE CHECKSUM
 #=================================================
 
 # Calculate and store the config file checksum into the app settings
-ynh_store_file_checksum --file="$app_service_registration_path/$app.yaml"
-ynh_store_file_checksum --file="$mautrix_config_path"
+ynh_store_file_checksum --file="/etc/matrix-$synapse_instance/app-service/$app.yaml"
+ynh_store_file_checksum --file="$final_path/config.yaml"
 
 #=================================================
 # GENERIC FINALIZATION
-#=================================================
-# SECURE FILES AND DIRECTORIES
-#=================================================
-
-### For security reason, any app should set the permissions to root: before anything else.
-### Then, if write authorization is needed, any access should be given only to directories
-### that really need such authorization.
-
-# Set permissions to app files
-chown -R root: $final_path
-#chown -R root: $signald_path
-
-# WARNING : theses command are used in INSTALL, UPGRADE, RESTORE
-# For any update do it in all files
-chown $mautrix_bridge_user:root -R $final_path
-#chown $signald_user:root -R $signald_path
-
 #=================================================
 # SETUP LOGROTATE
 #=================================================
 ynh_script_progression --message="Configuring log rotation..." --weight=3
 
 # Use logrotate to manage application logfile(s)
-ynh_use_logrotate --logfile "$log_filename"
-chown $mautrix_bridge_user:root -R /var/log/$app
+ynh_use_logrotate
+chmod -R 600 "/var/log/$app"
+chmod 700 "/var/log/$app"
+chown -R $app:$app /var/log/$app
 
 #=================================================
 # ADVERTISE SERVICE IN ADMIN PANEL
 #=================================================
-#yunohost service add $app --log "/var/log/$app/log.log"
-# if using yunohost version 3.2 or more in the 'manifest.json', a description can be added
-yunohost service add $app --description "$app daemon for bridging Signal and Matrix messages" --log "$log_filename"
-#yunohost service add $signald_name --description "$signald_name daemon for Signal messages" --log "$log_filename"
+ynh_script_progression --message="Integrating service in YunoHost..." --weight=1
+
+yunohost service add $app --description="$app daemon for bridging Whatsapp and Matrix messages" --log "/var/log/$app/$app.log"
 
 #=================================================
 # START SYSTEMD SERVICE
 #=================================================
-ynh_script_progression --message="Starting a systemd service..." --weight=1
+ynh_script_progression --message="Starting a systemd service..." --weight=15
 
 # Start a systemd service
-#ynh_systemd_action --service_name=$signald_name --action="start"
-#sleep 5
-ynh_systemd_action --service_name=$app --action="start"
+ynh_systemd_action --service_name=$app --action="start" --log_path="/var/log/$app/$app.log"
 # Wait until the synapse user is created
 sleep 30
```

Comment:
BAD! Racy, could take longer on slow systems, and unecessarily slow. Should rather check that the user exists with  psql, and move that sleep to the condition to avoid waiting if not necessary.